.github/workflows/msbuild.yml

```
@@ -2,7 +2,7 @@ name: MSBuild
 
 on:
   pull_request_target:
-    branches: [ master ]
+    branches: [ dev-workflow-sonar ]
```

Надо исправить опечатку, оставшуюся от тестов:
```suggestion
    branches: [ master ]
```